pkg/chartverifier/checks/helm.go

```diff
@@ -200,27 +201,39 @@ func getImageReferences(chartUri string, vals map[string]interface{}) ([]string,
 	mem.SetNamespace("TestNamespace")
 	actionConfig.Releases = storage.Init(mem)
 
-	imagesMap := make(map[string]bool)
-
 	txt, err := actions.RenderManifests("testRelease", chartUri, vals, actionConfig)
+	if err != nil {
+		return nil, err
```

Comment:
Wrap the error, so we know where the error comes from:

```go
return nil, fmt.Errorf("error rendering manifests: %w", err)
```

Tagging the error allow us to easily find the source of issue in a debugging session.